packages/httpsnippet-client-api/src/index.ts

```diff
@@ -47,9 +44,8 @@ function getAuthSources(operation: Operation) {
     return matchers;
   }
 
-  const security = operation.prepareSecurity();
-  Object.keys(security).forEach((id: SecurityType) => {
-    security[id].forEach(scheme => {
+  Object.entries(operation.prepareSecurity()).forEach(([, schemes]) => {
```

Comment:
Rewrote this whole loop because I don't know what I was smoking when I did that `object.keys()` mess earlier. 🪴